src/resolvers/unixfs-v1/index.js

```diff
@@ -0,0 +1,81 @@
+'use strict'
+
+const errCode = require('err-code')
+const UnixFS = require('ipfs-unixfs')
+const findShardCid = require('../../utils/find-cid-in-shard')
+
+const findLinkCid = (node, name) => {
+  const link = node.links.find(link => link.name === name)
+
+  return link && link.cid
+}
+
+const contentExporters = {
+  raw: require('./content/file'),
+  file: require('./content/file'),
+  directory: require('./content/directory'),
+  'hamt-sharded-directory': require('./content/hamt-sharded-directory'),
+  metadata: (cid, node, unixfs, ipld) => {},
+  symlink: (cid, node, unixfs, ipld) => {}
```

Comment:
`throw new Error('Not implemented')` for now?

Reply:
Done